backend/metaHandlers.ts

```diff
@@ -111,3 +112,27 @@ export const fetchChannelsSettings = async (
     ? prepareResponseFromMetadata(privateMetadata, channelSlugs, false)
     : null;
 };
+
+export const validateConfigurationBeforeSave = (
+  channelsConfiguration: ConfigurationPayload
+) => {
+  const activeWithoutApiKeys = Object.entries(channelsConfiguration)
+    .filter(
+      ([_, configuration]) =>
```

Comment:
I think you can even remove `_`
```suggestion
      ([, configuration]) =>
```